newIDE/app/src/EventsSheet/InstructionEditor/InstructionOrExpressionSelector/index.js

```diff
@@ -171,12 +174,9 @@ export default class InstructionOrExpressionSelector<
                   }) =>
                     renderInstructionOrExpressionListItem({
                       instructionOrExpressionMetadata: enumeratedInstructionOrExpressionMetadata,
-                      id:
-                        'instruction-or-expression-' +
-                        enumeratedInstructionOrExpressionMetadata.type.replace(
-                          /:/g,
-                          '-'
-                        ),
+                      id: getInstructionOrExpressionIdentifier(
```

Comment:
just to be sure, Isn't this ID used for the in-app tutorial?

Reply:
It is but not for instructions linked to object so it does not change their ids